client/my-sites/feature-upsell/ads-upsell.jsx

```diff
@@ -9,6 +9,7 @@ import React, { Component } from 'react';
 import Gridicon from 'gridicons';
 import { connect } from 'react-redux';
 import { localize } from 'i18n-calypso';
+import { compose } from 'redux';
```

Comment:
I believe everywhere we do this in Calypso, we use `lodash.flowRight`. Any reason to prefer this, or could we use the lodash version as a convention?

Reply:
@sirreal no problems with flowRight, I'll update it then. Does it look okay otherwise?

Reply:
@sirreal I just changed it to flowRight